extensions/notebook/src/book/bookTreeView.ts

```diff
@@ -430,23 +432,23 @@ export class BookTreeViewProvider implements vscode.TreeDataProvider<BookTreeIte
 		// the top we'll expand nodes until we find the parent of the Notebook we're looking for
 		// get the children of root node and expand the nodes to the notebook level.
 		await this.getChildren(parentBook.rootNode);
-		// The path to the parent of the Notebook we're looking for (this is the node we're looking to expand)
-		const parentPath = notebookPath.substring(0, notebookPath.lastIndexOf(path.posix.sep));
+		// The path to the Notebook we're looking for (these are the nodes we're looking to expand)
+		const notebookFolders = notebookPath.split('/');
```

Comment:
Why are you changing this logic anyways? This doesn't seem like it'd be directly related to the bug - unless this was functionally just broken to begin with.

Reply:
yeah, I was debugging a particular scenario where either conditions I had before returned a value and realized that there was a bug. So I changed it.

Reply:
Was the bug the thing causing the problem in the linked issue? Or was this just completely broken and not working at all?